create_project.yml

```diff
@@ -10,22 +10,24 @@
 
   tasks:
     - name: Display playbook name
-      debug: msg="==== Starting create_project playbook ===="
+      debug: msg="==== Starting create_namespace playbook ===="
       tags: deploy
 
     - import_tasks: tasks/set_vars.yml
 
-    - name: Make sure the project exists in OpenShift and is up-to-date
+    - name: Make sure the namespace exists in K8S and is up-to-date
       k8s:
         state: present
         definition:
-          apiVersion: project.openshift.io/v1
-          kind: Project
-          description: "{{ project_description | default(project_display_name) | default(project_name) }}"
-          displayName: "{{ project_display_name | default(project_name) }}"
+          apiVersion: v1
+          kind: Namespace
           metadata:
             name: "{{ project_name }}"
-      register: new_project
-      until: new_project.result is defined and new_project.result.status is success
+            labels: "{{ k8s_namespace_labels | default({}) | combine({ 'name': project_name }) }}"
```

Comment:
should you search and replace all occurrences of "project" including the name of this file? 
Probably yes as you did it for "delete"

Reply:
Should we ? I like the concept of «project» to refer to a specific customer/environment, that's why I kept the «project_name» variable and «create_project.yml» task. I find it confusing it to call it «namespace» since it's more than just a k8s entity.

WDYT ? It's just semantics, but we need to make a decision on this.

Reply:
well... I'm not sure and would like to hear @lunika and @jmaupetit on this but would find it confusing to keep the name project if it is a concept that does not exist in k8s and have 2 names to designate the same think.

Reply:
IMO the question here is: would a k8s user understand us if we talk about **projects** instead of **namespaces** (considering that this user is not aware of openshift's semantic)? If not, then it's probably a good idea to move to a namespace-based semantic.

Reply:
It seems you agree on this point and it's OK for me!
I propose to implement Samuel's suggestion (removing all occurences of "project") in another PR, because there are a lot of `{{ project_name }}` and I don't want to flood this PR with a lot of new changes. WDYT @sampaccoud @jmaupetit ?

Reply:
👍